snippets/set-textarea-value.js

```diff
@@ -0,0 +1,49 @@
+
+var selector = "<TODO: REPLACE>";
+var setTextArea = "<TODO: REPLACE>";  //TODO: テキストエリアに入力したい内容を指定してください。改行を含めたい場合には、「\n」を指定してください。
+
+/**
+ * --------- ここから下は変える必要はありません ----------
+ */
+
+/**
+ * 要素を探す
+ */
+var element = document.querySelector(selector);
+
+/**
+ * 要素がなければ処理を中断する
+ */
+if (!element) {
+  throw new Error(
+    "Error: cannot find the element with selector(" + selector + ")."
+  );
+}
+
+/**
+ * 値を入力する
+ */
+changeTextArea(element, setTextArea);
+
+function changeTextArea(input, value) {
```

Comment:
nit: I assume the function's responsibility is to input a text. How about using the term instead of `change`?

```suggestion
function inputTextToTextArea(input, value) {
```